src/expressServer.js

```diff
@@ -17,6 +17,8 @@ const pathToSwaggerUi = require('swagger-ui-dist').absolutePath();
 const { readFileSync } = require('fs');
 const { join } = require('path');
 const expressStaticGzip = require('express-static-gzip');
+const recommendationRoutes = require('./routes/recommendationRoutes');
+const recommendationController = require('../controllers/recommendationController');
```

Comment:
can be removed per lint warning 👍

Reply:
I have removed line 21,  as route  line 20 is required on line 137 
`app.use('api/v1/recommendations', recommendationRoutes);`
commit a2c198d2a525d0bdd8623b71066d27ef1cef8e6d